flake.nix

```diff
@@ -219,6 +219,7 @@
 
         enableParallelBuilding = true;
 
+        configureFlags = testConfigureFlags; # otherwise configure fails
```

Comment:
I wonder if it might be better to instead allow disabling the unit tests without disabling the functional tests.

What do you think, @robert?

Reply:
(Probably meant to ping @roberth?)

Reply:
I think improving the granularity is probably good, but I don't think this PR's change affects the need for rebuilds in any significant way, so there's no hurry.